draft-irtf-cfrg-vdaf.md

```diff
@@ -3159,6 +3193,11 @@ way. The parameters for this circuit are summarized below.
 | `Field`          | `Field64` ({{fields}})       |
 {: title="Parameters of validity circuit Count."}
 
+| Parameter        | Value                        |
+|:-----------------|:-----------------------------|
+| `PROOFS`         | `1`                          |
+{: title="Values for Prio3 specific parameters."}
```

Comment:
As I mentioned above, I don't think we should create a table for just one parameter. Idea: Perhaps add the operational parameters (I believe just `Xof` and the `Flp`) to this table as well? Here and below.